frontend/src/store/auth/index.js

```diff
@@ -18,10 +23,12 @@ const state = {
 const getters = {
 // logged into firebase (authenticated account)
   isFirebaseAuthorised: ({ userAuthObject }) => !!userAuthObject,
+
+  emailVerified: ({ userAuthObject }) => !!userAuthObject && userAuthObject.emailVerified,
   // TODO not sure if this is useful...
   hasProfile: ({ profile }) => !!profile,
-  // logged into backend (existing profile) and authed with firebase
-  isLoggedIn: ({ profile, userAuthObject }) => !!profile && !!userAuthObject,
+  // logged into backend (existing profile) and authed with firebase with a verified email address
+  isLoggedIn: ({ profile, userAuthObject }) => !!profile && !!userAuthObject && userAuthObject.emailVerified,
```

Comment:
have noticed recently that:

```js
!!{} = true
```

Reply:
Yeah... That shouldn't break stuff here. But maybe it would be better to change these to `thing === null` to be more explicit